README.md

```diff
@@ -34,7 +34,23 @@ Supports popular public checkpoints such as:
 
 This implementation extends on [Lit-LLaMA](https://github.com/lightning-AI/lit-llama) and [nanoGPT](https://github.com/karpathy/nanoGPT), and it's **powered by [Lightning Fabric](https://lightning.ai/docs/fabric/stable/) ⚡**.
 
-## Design principles
+
+
+---
+
+**NeurIPS 2023 Large Language Model Efficiency Challenge: 1 LLM + 1GPU + 1Day**
+
+Lit-GPT is the official starter kit of the [NeurIPS 2023 LLM efficiency challenge](https://llm-efficiency-challenge.github.io), which is focused on tackling the three main challenges of LLMs: 1) transparency, 2) standard benchmarks, and 3) hardware access. 
```

Comment:
These three bullet points are too concise in my opinion.
Of course anyone who is interested can read about it more on the challenge's page, but first this person needs to be interested 😄.
So maybe just copy what is written on NeurIPS page?

   1) Lack of transparency around model training methods leads to a majority of models being not reproducible.
   2) The absence of a standard benchmark to evaluate these models side-by-side.
   3) Insufficient access to dedicated hardware prevents widespread availability and usage of these models.

Reply:
Thanks for the feedback! On a second thought, I removed these three points and added something about the base rules/requirements. I really wanted to be short here so that it doesn't take up too much space in the Readme. I would still think that the Readme should convey what Lit-GPT is. The NeurIPS challenge is a nice application of it, but I didn't want it too take up too much screen estate. 
But I am open to feedback in case the others have some thoughts as well!

Reply:
> I would still think that the Readme should convey what Lit-GPT is.

Totally agree with you on that note. At the same time the info about the challenge will not stay for too long so, I guess, it's acceptable to make it more visible.

At the same time I think we are spending too much time on it. So maybe let's stick to your variant and after, if necessary, someone can add something to highlight it.

Reply:
As a side-note: maybe it's better to use a some nice icon?
This one I took from [here](https://www.flaticon.com/free-icon/challenge_3310330).
Not sure about licensing though...

---
![challenge](https://github.com/Lightning-AI/lit-gpt/assets/58434077/d5ca145a-760a-4c67-81d4-1953fb9457a6) **NeurIPS 2023 Large Language Model Efficiency Challenge: 1 LLM + 1 GPU + 1 Day**

Lit-GPT is the official starter kit of the [NeurIPS 2023 LLM efficiency challenge](https://llm-efficiency-challenge.github.io), which is competition centered around training a foundational (base) LLM for 24h on a single A100 or 4090.

If you are interested in participating, you can learn more about the official NeurIPS LLM efficiency challenge website [here](https://llm-efficiency-challenge.github.io). **The submission deadline is Oct 15th, 2023.**

---